config/api.yml

```diff
@@ -948,6 +948,16 @@
       - :name: delete
       :delete:
       - :name: delete
+  :custom_button_events:
+    :description: Custom Button Events
+    :options:
+    - :subcollection
+    :verbs: *g
+    :klass: CustomButtonEvent
+    :subcollection_actions:
```

Comment:
Do we also need `subresource_actions` for `get`?

Reply:
I'm not completely familiar what that would allow us, can you please help me?

Reply:
IIUC, `subcollection_actions` `get` allows `get /api/users/:id/custom_button_events`, but you would need `subresource_actions` `get` to allow `get /api/users/:id/custom_button_events/:id`.  Is that correct @abellotti?

Reply:
@bdunne we don't need those, this feature is an admin-only debug thing